appshell/appshell_extensions_platform.cpp

```diff
@@ -7,6 +7,7 @@
 #include "appshell/browser/main_context.h"
 #include "appshell/browser/root_window_manager.h"
 #include "appshell/browser/root_window_gtk.h"
+#include <unicode/unistr.h>
```

Comment:
Do these headers work for all linux versions?

Reply:
Yes, I have just resolved indirect header dependency in this case.
These are standard unicode library headers.